### ISSUE TITLE
bulk upload recent history (50 items?)

### ISSUE THREAD

**Issue by jaredhirsch** [mozilla/chronicle-addon#11]
Once mozilla/chronicle#217 lands, we will want to bulk upload the user's history as a way to quickly create an awesome experience for new users.
1. Most recent 20-50 items: upload with regular (or higher-than-normal) priority, so that users see their feed populate quickly.
2. Older items: upload with lower priority (this will destroy our embedly key, maybe this can be split into a separate bug or conditionally enabled using a flag)
